lld/MachO/Driver.cpp

```diff
@@ -349,13 +352,19 @@ void multiThreadedPageInBackground(DeferredFiles &deferred) {
       return;
 #ifndef NDEBUG
     totalBytes += buff.size();
-    numDeferedFilesTouched += 1;
+    numDeferedFilesAdvised += 1;
 #endif
 
+#if _WIN32
```

Comment:
I would put all this in a API declared in `llvm/include/llvm/Support/Memory.h` and defined in `llvm/lib/Support/Windows/Memory.inc` for Windows and `llvm/lib/Support/Unix/Memory.inc` for Unix/MacOS. Perhaps something like `Memory::prefetchMappedMemory` to be in line with the other APIs there.

Reply:
I agree but I'm sorry but I'm haven't signed on for larger refactors.